flink-tests/src/test/java/org/apache/flink/test/streaming/runtime/SinkMetricsITCase.java

```diff
@@ -147,7 +147,7 @@ private void assertSinkMetrics(
                                             * MetricWriter.RECORD_SIZE_IN_BYTES)));
             // MetricWriter is just incrementing errors every even record
             assertThat(
-                    metrics.get(MetricNames.NUM_RECORDS_OUT_ERRORS),
+                    metrics.get(MetricNames.NUM_RECORDS_SEND_ERRORS),
```

Comment:
Nit: Since you only deprecated the old metric it probably makes sense to also verify the deprecated metric.